drivers/entropy/entropy_bt_hci.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021, Commonwealth Scientific and Industrial Research
+ * Organisation (CSIRO) ABN 41 687 119 230.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT zephyr_bt_hci_entropy
+
+#include <drivers/entropy.h>
+#include <bluetooth/hci.h>
+#include <string.h>
+
+static int entropy_bt_init(const struct device *dev)
+{
+	/* Nothing to do */
+	return 0;
+}
+
+static int entropy_bt_get_entropy(const struct device *dev,
+				  uint8_t *buffer, uint16_t length)
+{
+	struct bt_hci_rp_le_rand *rp;
+	struct net_buf *rsp;
+	int req, ret;
+
```

Comment:
Please check if you are in ISR, since you cannot send HCI commands from interrupt context.

Reply:
Is this required given the existence of the `get_entropy_isr` API function, which this driver does not implement?

Reply:
That's fair, let's leave it here.